internal/tss/utils/transaction_service.go

```diff
@@ -0,0 +1,240 @@
+package utils
```

Comment:
I've kept things the way they are for now - ie: everything under the tss directory. I did this because we'd have to make tss subdirectories inside of internal/services and also internal/data, because for tss, I'm abstracting away the storage layer as an interface as well. If we feel differently after another discussion, I am down to change this